compose/foundation/foundation/src/jsWasmMain/kotlin/androidx/compose/foundation/text/KeyMapping.web.kt

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
This is not a new file. It was renamed from `KeyMapping.js.kt` + updated for clipboard related key events